org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/wizard/template/AbstractFileTemplate.java

```diff
@@ -38,7 +38,7 @@ protected String getName() {
 		return info.getName();
 	}
 
-	void setTemplateInfo(TemplateFileInfo info) {
+	protected void setTemplateInfo(TemplateFileInfo info) {
```

Comment:
@ArneDeutsch can you add `@since 2.25` to the newly protected/public methods?

Reply:
@ArneDeutsch I also miss the `@since 2.25` tags, otherwise LGTM.

Reply:
Added @since 2.25 to all newly protected methods.